client/src/Results/index.js

```diff
@@ -1,15 +1,47 @@
 import React, { Component } from "react";
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import Button from 'material-ui/Button';
+import Paper from 'material-ui/Paper';
 
 class Results extends Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      results: props.results || [{ name: "CMPSC 40", times: "M W 1:00 p.m - 1:50 p.m" }]

```

Comment:
This could be rewritten as a stateless component since the state never changes. It renders a lot faster.

Reply:
Sure, we can make that change in a separate PR. I would argue that there will be a need for state later on in the Results Component. I used state here because I wanted to make my changes scoped outside of the HOC for the PR.